### PR TITLE
set version in session search

### PR DESCRIPTION
**Pull request by Ximura**
In case server and client has not default value in gm_version cvar (not 1), client couldn't find and connect to server

Client always search for server with version **1**, even when his gm_version is different

I suggest use VF_CONST_CVAR for **gm_version** to prevent client from changing it (ex. by adding it to config)

### CODE REVIEW

File: dev/Gems/Multiplayer/Code/Source/MultiplayerCVars.cpp [hunk 1]
```diff
@@ -588,7 +588,7 @@ namespace Multiplayer
         carrierDesc.m_enableDisconnectDetection = !!gEnv->pConsole->GetCVar("gm_disconnectDetection")->GetIVal();
         carrierDesc.m_connectionTimeoutMS = 10000;
         carrierDesc.m_threadUpdateTimeMS = 30;
-        carrierDesc.m_version = gEnv->pConsole->GetCVar("gm_version")->GetIVal();
```
Comment (Ximura) [inline dev/Gems/Multiplayer/Code/Source/MultiplayerCVars.cpp]: we already set version in Multiplayer::Utils::InitCarrierDesc

**Comment by AMZN-puvvadar** [comment]
Hi @Ximura, 

Would you be able to resubmit your diff? The current diff appears to have pulled in a multitude of irrelevant files (css changes etc.) that I imagine were not intended to be part of the change. In its current state the diff cannot be pulled in for review. Thanks!

**Comment by Ximura** [comment]
Hi @AMZN-puvvadar 
You can check diff here
https://github.com/fraglab/Lumberyard/commit/b40a9d4c0b74598a2be806148d999d44121cf248

**Comment by AMZN-puvvadar** [comment]
Hi @Ximura,

I'm pleased to report that we've accepted this changelist. Thanks for contributing to Lumberyard!

**Pull request closed without merging**